homeassistant/components/hunterdouglas_powerview/__init__.py

```diff
@@ -97,6 +99,7 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
 
     coordinator = PowerviewShadeUpdateCoordinator(hass, shades, hub_address)
     coordinator.async_set_updated_data(PowerviewShadeData())
+
     hass.data.setdefault(DOMAIN, {})[entry.entry_id] = {
```

Comment:
In a future PR we should create a dataclass in a `models.py` instead of using a `dict` here